examples/consume.ts

```diff
@@ -0,0 +1,95 @@
+import Fluvio, {
+    FetchLogOption,
+    FetchOffset,
```

Comment:
We updated some terminology about these in the offset-api PR. `FetchOffset` is now just `Offset`, and `FetchLogOption` was changed to `ConsumerConfig` (since it's used in both fetching and streaming)